unstructured/partition/pdf.py

```diff
@@ -282,14 +288,27 @@ def _partition_pdf_or_image_local(
             pdf_image_dpi=pdf_image_dpi,
         )
 
+        extracted_layout = []
         if pdf_text_extractable is True:
             # NOTE(christine): merged_document_layout = extracted_layout + inferred_layout
-            merged_document_layout = process_file_with_pdfminer(
-                inferred_document_layout,
-                filename,
+            extracted_layout = process_file_with_pdfminer(
+                filename=filename,
+                dpi=pdf_image_dpi,
             )
```

Comment:
This could be:
```python
extracted_layout = (
    process_file_with_pdfminer(filename=filename, dpi=pdf_image_dpi)
    if pdf_text_extractable
    else []
)
```

which avoids having to track a mutable variable and is more direct. Also not sure if comment is still relevant.